Frends.OAuth.ParseToken/Frends.OAuth.ParseToken.Tests/UnitTests.cs

```diff
@@ -1,72 +1,126 @@
-using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Frends.OAuth.ParseToken.Definitions;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Newtonsoft.Json.Linq;
 
 namespace Frends.OAuth.ParseToken.Tests;
 
 [TestClass]
 public class UnitTests
 {
-    private static readonly string AuthHeader = File.ReadAllText(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "../../../Files/AuthHeader.txt"));
+    private static readonly string _authHeader = File.ReadAllText(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "../../../Files/AuthHeader.txt"));
     readonly JObject JwkKeys = JObject.Parse(File.ReadAllText(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "../../../Files/JwkKeys.json")));
+    private static Input _input = new();
+    private static Options _options = new();
 
-    Input? input;
-    Options? options;
-
-    /// <summary>
-    /// Parse token with WellKnownConfigurationUrl.
-    /// </summary>
-    [TestMethod]
-    public async Task TokenWithWellKnownUriTest()
+    [TestInitialize]
+    public void Init()
     {
-        input = new Input
+        _input = new Input()
         {
-            StaticJwksConfiguration = null,
-            Issuer = "https://frends.eu.auth0.com/",
             Audience = "fIVLouKUZihXfYP3tdO9D3dwd6ZNS9Be",
-            AuthHeaderOrToken = AuthHeader,
+            AuthHeaderOrToken = _authHeader,
             ConfigurationSource = ConfigurationSource.WellKnownConfigurationUrl,
+            Issuer = "https://frends.eu.auth0.com/",
+            StaticJwksConfiguration = null,
             WellKnownConfigurationUrl = "https://frends.eu.auth0.com/.well-known/openid-configuration"
         };
 
-        options = new Options
+        _options = new Options()
         {
             SkipAudienceValidation = false,
             DecryptionKey = null,
             SkipIssuerValidation = true,
             SkipLifetimeValidation = true,
             DecryptToken = false,
         };
+    }
 
-        var result = await OAuth.ParseToken(input, options, default);
+    [TestMethod]
+    public async Task ParseTokenTest_WithWellKnownUri()
+    {
+        var result = await OAuth.ParseToken(_input, _options, default);
         Assert.IsNotNull(result.SecurityKeyId != null || result.SigningKeyId != null);
```

Comment:
Please use `IsTrue` instead, since OR operation always returns a boolean and never null, which means that this Assert is always true.